index.html

```diff
@@ -88,63 +88,63 @@ <h4 class="services__heading">Customization</h4>
         </ul>
       </div>
     </section>
-    <section class="container">
-      <h5 class="category__description__title">EASY AND FAST</h5>
```

Comment:
Używasz klas z innej sekcji i trochę to może być mylące, moja sugestia to utworzyć nową klasę nawiązującą do tej sekcji i dołożyć klasę z odpowiednią czcionką.

Reply:
Tutaj użyłem klasy, którą stworzyłeś u siebie. Jak Adam wspominał, jak najwięcej klas powinno być reużywalnych.

Reply:
To powinna być uniwersalna klasa, trzeba zmienić nazewnictwo.

Reply:
done